packages/docs/pages/GettingStarted/GettingStartedPage.tsx

```diff
@@ -16,99 +16,95 @@ const GettingStartedPage = () => {
       </figure>
 
       <FlexItem alignSelf="center">
-        <H1>BigDesign Developer Playground</H1>
+        <H3>BigDesign Developer Playground</H3>
```

Comment:
We need to keep the headings in proper order. Jumping from nothing to an `<h3 />` breaks accessibility patterns.

Reply:
Agreed, fixed now. I actually didn't look at that page properly before